rastervision/data/raster_source/zxy2geotiff.py

```diff
@@ -0,0 +1,182 @@
+import tempfile
```

Comment:
I understand why this is here (it'll be used by a raster source), but I think this would be more discoverable if it was moved to the `rastervision.utils` package. I can see these methods being imported as a utility by other code that just wants to grab geotiffs from zxy endpoints.